scripts/primarybeamimage-fits.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+import argparse
+import logging
+import os
+
+import katsdpimageutils.primary_beam_correction as pbc
+
+
+def initialize_logs():
+    """
+    Initialize the log settings
+    """
+    logging.basicConfig(format='%(message)s', level=logging.INFO)
+
+
+def create_parser():
+    parser = argparse.ArgumentParser("Input a MeerKAT SDP pipeline continuum image and produce "
+                                     "primary beam corrected image.")
+    parser.add_argument('input',
+                        help='MeerKAT continuum uncorrected primary beam fits file')
+    parser.add_argument('--dir',
```

Comment:
It looks like this takes a filename rather than a directory, so the name is confusing. Rather call it something like `--output`. I haven't tried it, but I think you could also make it an optional positional argument by calling it just `'output'` and adding `nargs='?'`. Then users would run either `primarybeamimage-fits.py myinput.fits` (which computes an output filename) or `primarybeamimage-fits.py myinput.fits myoutput.fits` to specify the output filename.

Reply:
Noted and fixed.